components/crud-web-apps/jupyter/manifests/base/configs/spawner_ui_config.yaml

```diff
@@ -0,0 +1,186 @@
+# Configuration file for the Jupyter UI.
```

Comment:
Is it possible to change this file to a shortcut pointing to `components/crud-web-apps/jupyter/backend/apps/common/yaml/spawner_ui_config.yaml`? Would that work during installation or when pulling the files into the manifests repo?

Reply:
The plan is to copy `components/crud-web-apps/jupyter/manifests` into manifests, so I don't think it would work.
However, I would suggest to leave it out of scope for this PR.

Reply:
I think it is possible to create shortcuts in GitHub. Then, when you try and download a file shortcut it will grab the file it links to. Might make things easier in the future and avoid one version lagging behind the other.